test/tests.ts

```diff
@@ -204,6 +204,27 @@ describe('io-ts-promise', () => {
       return expect(tPromise.decode(type, value)).to.eventually.equal(value);
     });
 
+    it('resolves promise on falsy string', () => {
+      const type = t.string;
+      const value = '';
+
+      return expect(tPromise.decode(type, value)).to.eventually.equal(value);
+    });
+
+    it('resolves promise on falsy boolean', () => {
```

Comment:
Thanks for writing tests for the bug as well. If you don't mind the additional effort, it might be good to include `null` and `undefined` test cases as well. I think the `undefined` case will fail with the current implementation.

Reply:
No problem. `undefined` seems to work, added the tests any way.

Reply:
Ah, true. It will work since of course `arguments[2]` does return `undefined`. Good to have the tests nonetheless.